### ISSUE TITLE
Xamarin and Xeeny

### ISSUE THREAD

**Issue by nexusarx** [MhAllan/Xeeny#15]
Hi , 

First of all , this is a fantastic solution . I tried to use for xamarin (Android - Windows) communication. It almost worked. I can connect from adnroid (Tcp and http) to a windows host , but the methode call does not work. When i call a function from the client it will run on the server , but never return on the client and it will be freez.  
I used " async await " methode call on xamarin. 

Did you try it ever ? 

Thank you ,
Karesz

**Comment by MhAllan** [comment]
yes It is tested on Android and Windows, could you please post your code
